initDB.js

```diff
@@ -0,0 +1,91 @@
+var config = require("config");
```

Comment:
Do we need this file? If yes shuold we move it to some folder, because I think we don't need it on project root level